content/docs/react-without-jsx.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: react-without-jsx
-title: React Without JSX
+title: JSX के बिना React
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+React का उपयोग करने के लिए JSX की आवश्यकता नहीं है। JSX के बिना React का उपयोग करना विशेष रूप से तब सुविधाजनक होता है जब आप अपने बील्ड परिवेश मे कंपाइलेशन को सेटअप नहीं करना चाहते हैं।
```

Comment:
should we add build and environment to words that shouldn't be translated? @arshadkazmi42 ?

Reply:
Actually makes sense. Or may be we can add it in hinglish `बिल्ड`  `एनवायरनमेंट`